File: Jhakaas - AndriodService/Jhakaas/src/com/adp/jhakaas/Lead.java
package com.adp.jhakaas;

import java.io.Serializable;

public class Lead implements Serializable{
	
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	private int id;
	private String firstName;
	private String lastName;
	private String avatar;
	
	public Lead(int id, String fName, String lName, String avatar){
		this.id = id;
		this.firstName = fName;
		this.lastName = lName;
		this.avatar = avatar;
	}

	public int getLeadId() {
		return id;
	}
	
	public String getFirstName() {
		return firstName;
	}

	public String getLastName() {
		return lastName;
	}

	public String getAvatar() {
		return avatar;
	}
}

File: Jhakaas - API/Models/Lead.cs

namespace Jhakaas___API.Models
{
    public class Lead
    {
        public int Id { get; set; }

        public string FirstName { get; set; }
        public string MiddleName { get; set; }
        public string LastName { get; set; }

        public string Email { get; set; }
        public long Phone { get; set; }

        public string Avatar { get; set; }
    }
}